src/d3kit-gridmap.js

```diff
@@ -6,6 +6,7 @@ var DEFAULT_OPTIONS = {
   initialWidth: 500,
   initialHeight: 340,
   keyFn: undefined,
+  durationFn: function() {return 250;},
```

Comment:
What I meant is `.duration()` can take either *value* or *function*, since the default is a constant value (250), you can just pass the value without wrapping it in a function. When you use it, you can still set `transitionDuration` to be your function. 

From D3's doc
> ##### transition.duration([duration]) 
> Specifies per-element duration in milliseconds. If duration is a constant, then all elements are given the same duration; otherwise, if duration is a function, then the function is evaluated for each selected element (in order), being passed the current datum d and the current index i, with the this context as the current DOM element. The function's return value is then used to set each element's duration. The default duration is 250ms. If duration is not specified, returns the duration bound to the first non-null element in the transition.